spring-cloud-dataflow-server/docker-compose.yml

```diff
@@ -54,7 +54,7 @@ services:
       - ZOOKEEPER_CLIENT_PORT=2181
 
   dataflow-server:
-    image: springcloud/spring-cloud-dataflow-server:${DATAFLOW_VERSION:?DATAFLOW_VERSION is not set!}
```

Comment:
The `latest` is not a valid image tag nether for the SCDF or Skipper server nor for the Stream or Task apps.  An explicit app version tags is required.